src/main/java/uk/gov/justice/digital/delius/jpa/standard/repository/ContactRepository.java

```diff
@@ -1,8 +1,21 @@
 package uk.gov.justice.digital.delius.jpa.standard.repository;
 
+import java.time.LocalDate;
+import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import uk.gov.justice.digital.delius.jpa.standard.entity.Contact;
 
 public interface ContactRepository extends JpaRepository<Contact, Long>, JpaSpecificationExecutor<Contact> {
+
+    @Query("SELECT contact FROM Contact contact "
+        + "WHERE contact.offenderId = :offenderId "
+        + "AND contact.event.eventId = :eventId "
+        + "AND contact.contactDate <= :toDate "
+        + "AND contact.enforcement = '1'")
```

Comment:
I would strongly recommend testing this is pre-prod and  check this is correct. i.e. will this only bring back "attendance" records (whatever that means)

Reply:
There's an action to review with Ed next week but as it stands this is what the ticket is saying